src/app/apis/details.js

```diff
@@ -0,0 +1,44 @@
+const LOAD = 'reactcapstone/apis/LOAD';
+
```

Comment:
- [ ] According to the requirements, you are supposed to provide a video which meets the following
  -  presenting the features of the project.
  - The maximum length of the video is 5 minutes.
  - also, The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
  - The video contains a demo of the project, highlighting all the project features.
  - Lastly, The video contains a highlight about a piece of the code the student finds interesting or is proud of making.